packages/core/src/awsService/ec2/utils.ts

```diff
@@ -42,6 +43,7 @@ export function getEc2SsmEnv(
             STREAM_URL: session.StreamUrl,
             SESSION_ID: session.SessionId,
             TOKEN: session.TokenValue,
+            LOG_LEVEL: globals.logOutputChannel.logLevel,
```

Comment:
looks like getLogger doesn't have e.g. `getLogLevel`. otoh, we could make the flag a simple boolean like `LOG_DEBUG`:
```suggestion
            LOG_DEBUG: getLogger().logLevelEnabled('debug') ? '1': '0',
```

that also would avoid the use of `-le` in the bash script

Reply:
I don't think `logLevelEnabled` works as intended. I was able to produce this: 
``` 
2024-12-05 17:11:27.838 [info] vscode log level: info
2024-12-05 17:11:27.838 [info] is debug enabled?: true
``` 
and I believe its because the internal log level is `debug` until the user changes it based on the code here: 
https://github.com/aws/aws-toolkit-vscode/blob/aa332da854c9d17ba258b459c622500911b1b0d9/packages/core/src/shared/logger/activation.ts#L43-L49. 

Is there something special about the fact that I am running locally? Otherwise this seems like a bug.

Reply:
I think when I added that "HACK" line, I assume that something in vscode would always overwrite the log-level, except when running in debug-mode. That line definitely needs some attention. 

Perhaps the easiest fix is to wrap that line in `if (!isRelease() && ...)` or whatever